test/unit/unit_tests_err.cpp

```diff
@@ -0,0 +1,73 @@
+
+#include <limits.h>
+#include "gtest/gtest.h"
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+#include "../../src/backends.h"
+#include "../../src/backends/torch.h"
+#include "../../src/config.h"
+#include "../../src/script_struct.h"
+#include "../../src/script.h"
+
+
+#ifdef __cplusplus
+}
+#endif
+
+
+namespace {
+
+    void Alloc_Reset() {
```

Comment:
It's ok have a 0-indent within a namespace.

Reply:
addressed in the previous commit